hi-ml-azure/src/health_azure/himl.py

```diff
@@ -340,8 +343,8 @@ def create_crossval_hyperparam_args_v2(
 
 def create_script_run(
     script_params: List[str],
-    snapshot_root_directory: Optional[Path] = None,
-    entry_script: Optional[PathOrString] = None,
+    snapshot_root_directory: Optional[Path],
```

Comment:
I wonder if this might have some side effects? 
especially for scripts that are not tested, I had a quick look at present scripts that are submitted to AML (tiles creation...), looks ok, but we might need to update docs

Reply:
It is not used across hi-ml, and anyway it should not be used because it's an internal method.